SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day3/ransom/ExerciseRansom.java

```diff
@@ -0,0 +1,55 @@
+package hackerrank.day3.ransom;
+
+import java.util.HashMap;
+import java.util.Scanner;
+
+/**
+ * @author Sebastian Herrera (bermeosebas@gmail.com)
+ * @created 31/08/21
+ */
+public class ExerciseRansom {
+
+    public static final String YES = "Yes";
+    public static final String NO = "No";
+
+    public static void getWordsMagazine() {
+
+        Scanner scanner = new Scanner(System.in);
+        int magazineSize = scanner.nextInt();
+        int noteSize = scanner.nextInt();
+
+        scanner.nextLine();
+        String[] magazine = scanner.nextLine().split(" ");
+        String[] note = scanner.nextLine().split(" ");
+
+        HashMap<String, Integer> frequentWord = new HashMap<>();
```

Comment:
Also remember Code for Interface, not for Implementation:
https://www.javaguides.net/2018/06/java-collection-framework-best-practices.html